code/processes/gateway.q

```diff
@@ -100,10 +100,16 @@ queryqueue:([queryid:`u#`long$()] time:`timestamp$(); clienth:`g#`int$(); query:
 // client details
 clients:([]time:`timestamp$(); clienth:`g#`int$(); user:`symbol$(); ip:`int$(); host:`symbol$())
 
+//Function to generate random placeholder
+genrand:{system ("S ",string `int$.z.T);-1?0Ng}
```

Comment:
It doesn't hugely matter given you are always using ~ for comparison, but could you change this to rand 0Ng? -1?0Ng will generate a 1 item list, an atom would be more appropriate.

Reply:
Understood. I'll fix this now!